src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -544,6 +545,14 @@ public void testDefaultPotionEffects()
 		}
 	}
 
+	@Test
+	public void testWarningStateAndShutdownMessage(){
+		final String SHUTDOWN_MESSAGE = "Shutting down server...";
+		Warning.WarningState defaultWarningState = Warning.WarningState.DEFAULT;
+
+		assertEquals(defaultWarningState, server.getWarningState());
+		assertEquals(SHUTDOWN_MESSAGE, server.getShutdownMessage());
+	}
```

Comment:
I don't think this Unit Test is really needed here.
In general Getter-testing is sometimes frowned upon, especially when they are constants since messing these up is not really likely to happen. And if the values should diverge one day, then that was probably intentional.

So to me this Unit Test is more of a maintenance burden than really helping to ensure that MockBukkit runs flawlessly.

Reply:
thanks for your suggestions. i'll leave it blank until new features come in.